docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+|    | 1.3 First tag created 13                                                                                                                                                                                                                                 |             | here a limit on how many tags users can create?                                    |
+| 13 | Text below created tag 1 line "Your first tag has been created" 2 line "Now you can add translated words to favorites using this tag"                                                                                                                    |             |                                                                                    |
+| 14 | Red button "create" is below all text, when pressed leads to create a tag page (12)                                                                                                                                                                      |             |                                                                                    |
+| 15 | There is no search text box                                                                                                                                                                                                                              |             |                                                                                    |
+|    | 1.4 Favorites_tags standard page 14                                                                                                                                                                                                                      |             |                                                                                    |
+| 16 | Chosen tag to add words to tag is highlighted by gray color                                                                                                                                                                                              |             |                                                                                    |
+| 17 | Search text box is located under "Favorites"                                                                                                                                                                                                             |             |                                                                                    |
+| 18 | placeholder text "Search" is written in search text box                                                                                                                                                                                                  |             |                                                                                    |
```

Comment:
Let's start each line with the capitalised letter but when you correct the UI element naming it will be ''Search' placeholder text...'